src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDisk.ps1

```diff
@@ -0,0 +1,201 @@
+    [CmdletBinding(DefaultParameterSetName='Disks_List')]
+    param(    
+        [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true, ParameterSetName = 'ResourceId_Disks_Get')]
+        [System.String]
+        $ResourceId,
+    
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true, ParameterSetName = 'InputObject_Disks_Get')]
+        [Microsoft.AzureStack.Management.Compute.Admin.Models.Disk]
+        $InputObject,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'Disks_List')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'Disks_Get')]
+        [System.String]
+        $Location,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'Disks_List')]
+        [System.Nullable`1[System.Int32]]
+        $Start = $null,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'Disks_List')]
+        [System.String]
+        $SharePath,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'Disks_List')]
+        [System.Nullable`1[System.Int32]]
+        $Count = $null,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'Disks_List')]
+        [System.String]
+        $UserSubscriptionId,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'Disks_List')]
+        [System.String]
+        $Status,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'Disks_Get')]
+        [Alias('DiskId')]
+        [System.String]
+        $Name
+    )
+
+    Begin 
+    {
+	    Initialize-PSSwaggerDependencies -Azure
+        $tracerObject = $null
+        if (('continue' -eq $DebugPreference) -or ('inquire' -eq $DebugPreference)) {
+            $oldDebugPreference = $global:DebugPreference
+			$global:DebugPreference = "continue"
+            $tracerObject = New-PSSwaggerClientTracing
+            Register-PSSwaggerClientTracing -TracerObject $tracerObject
+        }
+	}
+
+    Process {
+    
+    $ErrorActionPreference = 'Stop'
+
+    $NewServiceClient_params = @{
+        FullClientTypeName = 'Microsoft.AzureStack.Management.Compute.Admin.ComputeAdminClient'
+    }
+
+    $GlobalParameterHashtable = @{}
+    $NewServiceClient_params['GlobalParameterHashtable'] = $GlobalParameterHashtable
+     
+    $GlobalParameterHashtable['SubscriptionId'] = $null
+    if($PSBoundParameters.ContainsKey('SubscriptionId')) {
+        $GlobalParameterHashtable['SubscriptionId'] = $PSBoundParameters['SubscriptionId']
+    }
+
+    $ComputeAdminClient = New-ServiceClient @NewServiceClient_params
+
+    $DiskId = $Name
+
+ 
+    if('InputObject_Disks_Get' -eq $PsCmdlet.ParameterSetName -or 'ResourceId_Disks_Get' -eq $PsCmdlet.ParameterSetName) {
+        $GetArmResourceIdParameterValue_params = @{
+            IdTemplate = '/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/disks/{DiskId}'
+        }
+
+        if('ResourceId_Disks_Get' -eq $PsCmdlet.ParameterSetName) {
```

Comment:
Remove if/else and replace with body of if statement.

Reply:
OK